packages/@aws-cdk/integ-runner/lib/cli.ts

```diff
@@ -30,6 +30,7 @@ async function main() {
     .options('from-file', { type: 'string', desc: 'Read TEST names from a file (one TEST per line)' })
     .option('inspect-failures', { type: 'boolean', desc: 'Keep the integ test cloud assembly if a failure occurs for inspection', default: false })
     .option('disable-update-workflow', { type: 'boolean', default: false, desc: 'If this is "true" then the stack update workflow will be disabled' })
+    .option('app', { type: 'string', default: undefined, desc: 'The custom CLI command that will be used to run the test files. You can include {filePath} to specify where in the command the test file name should be inserted. Example: --app=\'"python3.8 {filePath}"\'. It is recommended to use single quotes to wrap the double quotes of the command into a literal string, to avoid parsing errors.' })
```

Comment:
```suggestion
    .option('app', { type: 'string', default: undefined, desc: 'The custom CLI command that will be used to run the test files. You can include {filePath} to specify where in the command the test file path should be inserted. Example: --app=\'"python3.8 {filePath}"\'. It is recommended to use single quotes to wrap the double quotes of the command into a literal string, to avoid parsing errors.' })
```